isb_lib/core.py

```diff
@@ -59,16 +62,18 @@ def relationAsSolrDoc(
     doc["tstamp"] = datetimeToSolrStr(ts)
     return doc
 
+
 def _shouldAddMetadataValueToSolrDoc(metadata: typing.Dict, key: typing.AnyStr) -> bool:
     shouldAdd = False
     value = metadata.get(key)
     if value is not None:
-        if key =='latitude':
-            shouldAdd = -90.0 <= value <= 90.0
+        if key == "latitude":
+            # Explicitly disallow bools as they'll pass the logical test otherwise and solr will choke downstream
+            shouldAdd = type(value) is not bool and -90.0 <= value <= 90.0
```

Comment:
One of the interesting things we found in the OpenContext data was that there was a longstanding bug where they had things like 
```
"latitude": False,
"longitude": False
```
 in lieu of omitting them from the data.  Interestingly, the previous check that made assertions about whether the lat/lon values were reasonable passed because `False` coerced down to 0 so it was between -90 and 90.  Had to tighten up the logic to omit Booleans from the test.  @ekansa is working on a fix in the OpenContext codebase but this workaround fixes it in the meantime.